source/core.py

```diff
@@ -45,6 +45,7 @@ class CallCancelled(Exception):
 postNvdaStartup = extensionPoints.Action()
 # inform those who want to know that NVDA has begun to exit.
 preNVDAExit = extensionPoints.Action()

```

Comment:
I think we can remove this extensionPoint if we wish now. Nobody would be using it unless it has been suddenly incorporated since the beta release. `handleNVDAModuleCleanupBeforeGUIExit` already calls `_terminate(globalPluginHandler)` which addons use for safe exit processes.